README.md

```diff
@@ -66,13 +64,8 @@ Look to the [defaults](defaults/main.yml) properties file to see the possible co
 ## Testing
 
 ```sh
-molecule test
-```
-
-or
-
-```sh
-./tests/scripts/run-test.sh
+$ pipenv install -r test-requirements.txt --python 2.7
```

Comment:
It would be nice if specifies that Python 3.x could be used 🙂

Reply:
Updated